src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -721,6 +729,14 @@ internal void ProcessFunctionMetadataResponses(FunctionMetadataResponse function
 
                     functionMetadata.SetFunctionId(metadata.FunctionId);
 
+                    foreach (var property in metadata.Properties)
+                    {
+                        if (!functionMetadata.Properties.TryAdd(property.Key, property.Value.ToString()))
+                        {
+                            _workerChannelLogger?.LogDebug("{metadataPropertyKey} is already a part of metadata properties", property.Key);
```

Comment:
Your original log had `functionId` as a property also - that does seem fairly useful to me. Do you think it is worth including?

Reply:
good point, let me include that as well